luigi/contrib/rdbms.py

```diff
@@ -288,6 +290,13 @@ class Query(luigi.task.MixinNaiveBulkComplete, luigi.Task):
     def host(self):
         return None
 
+    @property
+    def port(self):
+        """
+        Override to specify port separately from host.
```

Comment:
Should we be more clear here about `host:port` format?

Reply:
That was actually something very confusing to me. The abstract property here doesn't say anything about that format and basically leaves it to subclass (even worse the actual Target such as https://github.com/spotify/luigi/blob/198dfa1c20acdacd150cf7818f716c2dd31936ef/luigi/contrib/postgres.py#L128) to interpret. It feels a bit weird if we disclose the format here while don't somehow enforce it in the base class.

Reply:
Tried to "enforce" it in https://github.com/spotify/luigi/pull/2627/commits/0f77c2336039312263e4b8e4b98aaa1d51df726c